### PR TITLE
Don't check for Automated dispatch / ppa_soln_mode in Host Developer battery configs

### PR DESCRIPTION
**Pull request by mjprilliman**
## Description

Add a check for Host Developer financing configs on the Revenue page to not check for the interaction between automated dispatch and ppa_soln_mode

### CODE REVIEW

File: deploy/runtime/ui/Financial Solution Mode Single Owner.txt [hunk 1]
```diff
@@ -523,28 +523,29 @@ MultilineText
...
+2017
+// This form is a copy of Financial Solution Mode Single Owner
 // so ppa_soln_mode radio button can have different labels
-// to indicate IRR target is for the tax investor for flip 
-// structures. Any updates should be made to both forms.
+// to indicated IRR target is for the tax investor.
+// Any updates should be made to both forms.
 
-on_load{'Financial Solution Mode Single Owner'} = define()
+on_load{'Financial Solution Mode Flip Leaseback'} = define()
```
Comment (brtietz) [inline deploy/runtime/ui/Financial Solution Mode Single Owner.txt:532]: I'm a little confused by the name change here - should this match the top of the file?

**Review by brtietz — APPROVED** [review]
Fixes the problem that was reported, but I'm a little confused about the naming of the on_load function.

File: deploy/runtime/ui/Financial Solution Mode Single Owner.txt [hunk 2]
```diff
@@ -572,10 +573,6 @@ function display_message()
 	{
 		txt += 'IRR is the developer internal rate of return. ';
 	}
-	elseif( financing() == 'Community Solar' )
-	{
-		txt += 'Set PPA price or IRR target to zero if project only earns revenue from subscription payments on the Community Solar page.';
-	}
```
Comment (mjprilliman) [inline deploy/runtime/ui/Financial Solution Mode Single Owner.txt:578]: This should not have been changed on develop

File: deploy/runtime/ui/Financial Solution Mode Single Owner.txt (hunk 1 above)
Comment (mjprilliman) [reply deploy/runtime/ui/Financial Solution Mode Single Owner.txt:532]: You are correct this was changed on develop when it should not have been

**Review by brtietz — APPROVED** [review]
Looks great!

**Pull request merged**